en/services/parachute.md

```diff
@@ -0,0 +1,158 @@
+| 1024     | `PARACHUTE_ERROR_FLAGS_GS_CONNECTION_ERROR`  | There is an error with the connection between parachute and Ground Station (GS).                                 |
+| 2048     | `PARACHUTE_ERROR_FLAGS_GPS_ERROR`            | There is an error with the parachute's GPS.                                                                      |
+| 4096     | `PARACHUTE_ERROR_FLAGS_SUBSYSTEM_CONNECTION_ERROR` | There is an error with the connection between parachute and subsystem (e.g. remote controller, expansion board, etc.). |
+| 8192     | `PARACHUTE_ERROR_FLAGS_SUBSYSTEM_FW_ERROR`   | There is an error with the parachute subsystem firmware (e.g. wrong firmware version).                           |
+| 16384    | `PARACHUTE_ERROR_FLAGS_RESERVED_1`           | Reserved for future use.                                                                                         |
+| 32768    | `PARACHUTE_ERROR_FLAGS_RESERVED_2`           | Reserved for future use.                                                                                         |
+| 65536    | `PARACHUTE_ERROR_FLAGS_LOGGING_ERROR`        | There is an error with the parachute's internal logging system.                                                  |
+| 131072   | `PARACHUTE_ERROR_FLAGS_MODULE_RETIRED`       | This parachute module is retired (i.e. too many deployments).                                                    |
+| 262144   | `PARACHUTE_ERROR_FLAGS_GLOW_WIRE_ERROR`      | There is an error with the parachute glow wire.                                                                  |
+| 524288   | `PARACHUTE_ERROR_FLAGS_OFFBOARD_CONNECTION_ERROR` | There is an error with the MAVLink connection between parachute and offboard computer.                        |
+| 1048576  | `PARACHUTE_ERROR_FLAGS_IMU_CALIBRATION_ERROR`| Parachute's internal IMU calibration failed.                                                                     |
+
+
+#### MAV_CMD_SET_PARACHUTE_ARM
+This command can be used to arm/disarm the various parachute trigger sources. By configuring the arm/disarm setting of the various trigger sources, a user is able to change the behavior of the parachute when it is being used. The operation follows the normal [Command Protocol](https://github.com/mavlink/mavlink-devguide/blob/master/en/services/command.md) rules for command/acknowledgment.
+
+There are two parameters for this command. The first parameter is used to indicate the arm/disarm setting. Setting a bit to "0" indicated disarm and setting a bit to "1" indicates arm. The second parameter is a mask-byte that indicates which trigger source is being configured. The bits of the second parameter are mapped to [PARACHUTE_TRIGGER_FLAGS](#PARACHUTE_TRIGGER_FLAGS). The second parameter allows users to not have to track the status of all trigger sources when you want to set arm/disarm for specific sources. 
```

Comment:
1. Setting a bit to "0" ~~indicated~~ _indicates_ disarm
2. "The bits of the second parameter are mapped to `PARACHUTE_TRIGGER_FLAGS`" -> both parameters use `PARACHUTE_TRIGGER_FLAGS`

Reply:
also, mask-byte -> bitmask. And the last sentence reads kinda weird. Suggestion:
"The parachute will ignore all bit positions in the first parameter that are set to zero in the second parameter, which allows the user to granularly arm/disarm specific sources individually."